src/Sentry.NLog/Sentry.NLog.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Since the main SDK and NLog have targets also for .NET Framework, we can multi target here (and in the tests):
`<TargetFramework>netstandard2.0;net461</TargetFramework>`

[More context here](https://github.com/getsentry/sentry-dotnet/issues/135)